Apps/GatewayApi/src/appsettings.Development.json

```diff
@@ -56,5 +56,8 @@
         "Notifications": {
             "Enabled": true
         }
+    },
+    "EmailTemplate": {
+        "Host": "http://localhost:5002"
```

Comment:
Is it 5002 or 3025?

Reply:
It's 5002 for the site, 3025 for WebClient back-end.